flang/runtime/extensions.cpp

```diff
@@ -37,5 +75,19 @@ void FORTRAN_PROCEDURE_NAME(getarg)(
   (void)RTNAME(GetCommandArgument)(
       n, &value, nullptr, nullptr, __FILE__, __LINE__);
 }
+
+void FORTRAN_PROCEDURE_NAME(getlog)(std::int8_t *arg, std::int64_t length) {
+  std::array<char, LOGIN_NAME_MAX + 1> str;
+  int error = getlogin_r(str.data(), str.size());
+  assert(error == 0 && "getlogin_r returned an error");
```

Comment:
Prefer emitting a Fortran fatal runtime error here.

Can be done as:
```
Terminator terminator{__FILE__, __LINE__};
if (error != 0)
  terminator.Crash("getlogin_r returned error %d", error);
```
or simply
```
Terminator terminator{__FILE__, __LINE__};
RUNTIME_CHECK(terminator, error == 0);
```

Reply:
Done